js/containers/portal-dashboard/portal-dashboard-app.tsx

```diff
@@ -279,22 +279,23 @@ class PortalDashboardApp extends React.PureComponent<IProps, IState> {
   }
 
   private renderHeader = (assignmentName: string, headerViewMode: DashboardViewMode) => {
-    const { userName, setCompactReport, setHideFeedbackBadges, trackEvent } = this.props;
+    const { sequenceTree, userName, setCompactReport, setHideFeedbackBadges, trackEvent } = this.props;
     const { viewMode} = this.state;
     const color: ColorTheme = headerViewMode === "ProgressDashboard"
       ? "progress"
       : headerViewMode === "ResponseDetails" ? "response" : "feedback";
+
     return (
-      <Header
-        userName={userName}
-        setCompact={headerViewMode === "ProgressDashboard" ? setCompactReport : undefined}
-        setHideFeedbackBadges={headerViewMode === "ProgressDashboard" ? setHideFeedbackBadges : undefined}
-        assignmentName={assignmentName}
-        trackEvent={trackEvent}
-        setDashboardViewMode={this.setDashboardViewMode}
-        viewMode={viewMode}
-        colorTheme={color}
-      />
+      sequenceTree && <Header
```

Comment:
I'm not 100% sure about this particular instance, but I think if what comes after `&&` is made up of multiple lines, our convention is to have it start on a new line, as in line 175 above, right?